lib/Local.js

```diff
@@ -179,30 +184,27 @@ function Local(){
         break;
 
       case 'forceproxy':
+      case 'forceProxy':
         if(value)
-          this.forceProxyFlag = '-forceproxy';
-        break;
-
-      case 'hosts':
-        if(value)
-          this.hosts = value;
+          this.forceProxyFlag = '--force-proxy';
         break;
 
       case 'logfile':
+      case 'logFile':
         if(value)
           this.logfile = value;
         break;
 
-      case 'binarypath':
```

Comment:
The only reason I removed that was because I didn't see binarypath listed as an option [here](https://www.browserstack.com/local-testing), I assumed they removed that option. But I have no knowledge of the inner workings of the Browserstack tunnel, just going straight off of their documentation, so maybe that functionality still exists and it is undocumented.